escapeRegExp.js

```diff
@@ -22,7 +22,7 @@ const reHasRegExpChar = RegExp(reRegExpChar.source)
 function escapeRegExp(string) {
   return (string && reHasRegExpChar.test(string))
     ? string.replace(reRegExpChar, '\\$&')
-    : string
+    : string || ''
```

Comment:
can you wrap `string || ''` in parens please.

Reply:
> can you wrap string || '' in parens please.

Done